src/components/CategoryPicker.tsx

```diff
@@ -1,21 +1,34 @@
-import lodashGet from 'lodash/get';
-import React, {useMemo, useState} from 'react';
+import {useMemo, useState} from 'react';
 import {withOnyx} from 'react-native-onyx';
-import _ from 'underscore';
-import OptionsSelector from '@components/OptionsSelector';
+import type {OnyxEntry} from 'react-native-onyx';
 import useLocalize from '@hooks/useLocalize';
 import useThemeStyles from '@hooks/useThemeStyles';
 import * as OptionsListUtils from '@libs/OptionsListUtils';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
-import {defaultProps, propTypes} from './categoryPickerPropTypes';
+import type * as OnyxTypes from '@src/types/onyx';
+import {isEmptyObject} from '@src/types/utils/EmptyObject';
+import OptionsSelector from './OptionsSelector';
 
-function CategoryPicker({selectedCategory, policyCategories, policyRecentlyUsedCategories, onSubmit}) {
+type CategoryPickerOnyxProps = {
+    policyCategories: OnyxEntry<OnyxTypes.PolicyCategories>;
+    policyRecentlyUsedCategories: OnyxEntry<OnyxTypes.RecentlyUsedCategories>;
+};
+
+type CategoryPickerProps = CategoryPickerOnyxProps & {
+    /** It's used by withOnyx HOC */
+    // eslint-disable-next-line react/no-unused-prop-types
+    policyID: string;
+    selectedCategory: string;
+    onSubmit: () => void; // TO DO: Check if optional
+};
+
+function CategoryPicker({selectedCategory, policyCategories, policyRecentlyUsedCategories, onSubmit}: CategoryPickerProps) {
     const styles = useThemeStyles();
     const {translate} = useLocalize();
     const [searchValue, setSearchValue] = useState('');
 
-    const policyCategoriesCount = OptionsListUtils.getEnabledCategoriesCount(_.values(policyCategories));
+    const policyCategoriesCount = OptionsListUtils.getEnabledCategoriesCount(policyCategories ?? {});
```

Comment:
Object.values isn't needed anymore? Did you test it?

Reply:
Yes, `getEnabledCategoriesCount` uses `Object.values`